README.md

```diff
@@ -1,45 +1,67 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Webpack-template
+# To Do list
 
-> This is a ready webpack template that can be used when you start a new project.
+> Build a To Do List JavaScript application
+![app-screenshot](./app-screenshot.png)
 
-### Prerequisites
+## About The Project
+
+> The project I have built is based on a simple app which adds tasks, marks them as completed and deletes all completed tasks. This project has been built with the help of modern CSS3, Bootstrap 5, JavaScript and webpack to bundle JavaScript. Check the live demo to try.
+
+## Built With
+
+- JavaScript, HTML, CSS
+- Webpack
+
+## Getting Started
 
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
     - node
 
 ### Setup
-
-> Clone the repository to your local machine and run:
 ```
-npm install
-npm init -y
-npm install --save-dev webpack
-npm install --save lodash
-npx webpack --config webpack.config.js
-npm install webpack webpack-cli --save-dev
-npm install --save-dev style-loader css-loader
-npm install --save-dev csv-loader xml-loader
-npm install --save-dev html-webpack-plugin
-npm install --save-dev webpack-dev-server
+- npm install
+- Clone the repository to your local machine.
+- npm install webpack webpack-cli --save-dev
 ```
-
 ### Deployment
-
- > run:
+ run:
  ```
  npm run build
- npm start
  ```
+ and deploy the files in the ```dist``` folder
 
 ## Authors
 
 👤 **Ibrohim Rasulov**
 
-- [GitHub](https://github.com/IbrohimRasulov)
-- [Twitter](https://twitter.com/IbrohimRasu1ov)
-- [LinkedIn](https://www.linkedin.com/in/ibrohim-rasulov-a88352209/)
+- GitHub: [@IbrohimRasulov](https://github.com/IbrohimRasulov)
+- Twitter: [@IbrohimRasu1ov](https://twitter.com/IbrohimRasu1ov)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ibrohim-rasulov-a88352209/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
 
```

Comment:
- [ ] Kindly update the issue with a valid link to help users navigate easily and provide comments as you suggested. 👍